molgenis-app/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.1"
+services:
+    molgenis:
+      image: registry.molgenis.org/molgenis/molgenis-app:${TAG:-latest}
+      ports:
+        - "8081:8080"
+      environment:
+        - molgenis.home=/home/molgenis
+        - opencpu.uri.host=opencpu
+        - elasticsearch.transport.addresses=elasticsearch:9300
+        - db_uri=jdbc:postgresql://postgres/molgenis
+        - db_user=molgenis
+        - db_password=molgenis
```

Comment:
Hardcoded pasword required here?

Reply:
Is exposed internally only, only needs to match with the postgres password.